### PR TITLE
week2 homework

### PR DESCRIPTION
**Pull request by gcrk**
_If you had no problems you can just leave it empty_

> How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
around 3
> Was there anything that you struggled with?
the CSS part for the fan-site was very time consuming. / the "busy hand" was quite hard to finish
> Is there any particular code you want me to look into?

> Is there anything that you'd like some further information on?

> Roughly how long did it take?

### CODE REVIEW

**Comment by gcrk** [comment]
Update with the ATM exercise

How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
2
Was there anything that you struggled with?
not really
Is there any particular code you want me to look into?
Is there anything that you'd like some further information on?
Roughly how long did it take?
around 2H for the first working version. around 2H to improve some details

File: Guillaume Canneton/week02/atm/js/atm.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+let savBalance = 0;
+// Checking account part
+//first button - deposit on checking account
+$( document ).ready(function() {
+
+
+$('#checking-deposit').on('click', function(){
+  let $checkAmount = $( '#checking-amount' ).val();
+  if ( isNaN(parseInt( $checkAmount )) ){
+    window.alert("Please enter a number");//this statement to avoid a bug with NaN value
```
Comment (CamillaCdC) [inline Guillaume Canneton/week02/atm/js/atm.js:11]: Awesome!

File: Guillaume Canneton/week02/atm/js/atm.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  if ( savBalance <= 0 ){
+    $( '#savings-balance' ).addClass( 'zero' )
+  } else {
+    $( '#savings-balance' ).removeClass( 'zero' )
+  };
+  if ( checkBalance > 0 || savBalance > 0 ){
+    $( '#alert').css( 'visibility', 'hidden')
+  }
+}
+alertBalance(); //run the function to set the background at the first load of the page
```
Comment (CamillaCdC) [inline Guillaume Canneton/week02/atm/js/atm.js:89]: Great way to go about this!

**Comment by CamillaCdC** [comment]
CSS can be very time consuming! Involves a bit of trial and error - the console is great for this. Your CSS looks great! Nice work. 

Awesome work on the ATM homework. The comments throughout the code are great :)

**Pull request merged**